### PR TITLE
Task 4 - core - view a list of all articles

### PR DESCRIPTION
**Pull request by hessam-0**
FYI for anyone reviewing my code I have recently discovered style guides and switched away from using tabs for indentation. I now use double spaces. I also try to include all semicolons strictly as one guide mentioned inconsistency in this area leading to potential bugs, and I don't fancy discarding them all together. I am also exploring the other aspects like line length and nesting which I was always wary of, and whitespace. When I look at the commits on here I am noticing a bunch of inconsistencies which I miss in vscode!

### CODE REVIEW

**Comment by nc-mark-t** [comment]
I would recommend having a look at using prettier with the style guide configuration you want and then run it on your entire repo and use it before each PR

File: src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import { useState } from 'react'
 import './App.css'
+import ArticleList from './components/ArticleList'
 
 function App() {
   const [count, setCount] = useState(0)
 
   return (
     <>
```
Comment (nc-mark-t) [inline src/App.jsx:9]: This `<>` is unnecessary here

**Review by nc-mark-t — COMMENTED** [review]
Good work, you might want to add a header and in turn some navigation back to the home page as your app grows

**Comment by hessam-0** [comment]
> I would recommend having a look at using prettier with the style guide configuration you want and then run it on your entire repo and use it before each PR

Thansk for this mark, I am a bit of an idiot because I've had prettier installed for this entire time and did not realise I had to configure it to work, alwaays wondered what it was even doing, turns out in my case,  not a whole lot.

**Pull request merged**